docs/index.rst

```diff
@@ -1,39 +1,128 @@
 ################################
-Qiskit Experiments Documentation
+Qiskit Experiments documentation
 ################################
 
-About Qiskit Experiments
-========================
+.. warning::
+
+    This package is still under active development and it is very likely
+    that there will be breaking API changes in future releases.
+    If you encounter any bugs, please open an issue on
+    `Github <https://github.com/Qiskit/qiskit-experiments/issues>`_.
 
 Qiskit Experiments provides both a :doc:`library <apidocs/library>` of standard
 quantum characterization, calibration, and verification experiments, and a
 general :doc:`framework <apidocs/framework>` for implementing custom experiments
 which can be run on quantum devices through Qiskit.
 
-Experiments run on `IBMQ Provider <https://github.com/Qiskit/qiskit-ibmq-provider>`_
-backends can be stored and retrieved from an online experiment
-:doc:`database <apidocs/database_service>`.
+We've divided up the documentation into four sections with different purposes:
```

Comment:
For some reason apostrophes are not rendering nicely for me with this version of the docs. I have not noticed this with the current docs or with other qiskit docs.

![image](https://user-images.githubusercontent.com/1178780/219508747-414c9709-4e76-4266-b4c5-da2c69511c7e.png)

Reply:
Hmm, this is odd. That apostrophe is a plain one, not curly. Are all apostrophes garbled for you? It's fine on my local build and the artifact files from the docs CI build look okay to me too. I wonder if this has to do with the new version of `qiskit-sphinx-theme`.

Reply:
Yes, they all look garbled to me. I guess it is using some font that I don't have or something? I actually have to migrate to a new computer today, so I will see how it looks there. Do you know if any qiskit docs have been built recently? All the other ones I tried at random did not have garbled apostrophes.

Reply:
I got my new computer set up and built the docs on it. I am still seeing strange renderings in the docs. Based on the [qiskit_sphinx_theme README](https://github.com/Qiskit/qiskit_sphinx_theme), I think that the [Qiskit docs guide](https://qiskit.github.io/qiskit_sphinx_theme/index.html) is build using qiskit-sphinx-theme as well, and I don't see garbled apostrophes or other odd symbols there.

Reply:
Here is another example of some weird renderings I am seeing besides the apostrophes. I am seeing an odd A next to anchor links and the text versions of circuits included inside of `parsed-literal` blocks look very strange.

![image](https://user-images.githubusercontent.com/1178780/222547828-4f9a5aeb-cf36-4173-80f9-46e9cb42fd20.png)

Reply:
Will be solved in https://github.com/Qiskit/qiskit_sphinx_theme/pull/197.